File: docs/components_page/components/pagination/size.jl
using DashBootstrapComponents

pagination = html_div([
    html_div("Small"),
    dbc_pagination(max_value = 5, size = "sm"),
    html_div("Default"),
    dbc_pagination(max_value = 5),
    html_div("Large"),
    dbc_pagination(max_value = 5, size = "lg"),
])

File: docs/components_page/components/pagination/size.R
library(dashBootstrapComponents)

pagination <- div(
  list(
    div("Small"),
    dbcPagination(
      max_value = 5,
      size = "sm"
    ),
    div("Default"),
    dbcPagination(
      max_value = 5
    ),
    div("Large"),
    dbcPagination(
      max_value = 5,
      size = "lg"
    )
  )
)
